apps/condo/domains/miniapp/components/AppDescription/TopCard.tsx

```diff
@@ -0,0 +1,166 @@
I3/rEPXBNpaPWjwOVFXp0FKRiKZxupaHKFw2abgHuDuUhTJZBrdXY1504u5yGQlgvOhgh+LDoMEhnSAh0xZqRCeegO9PU3Q9XdOoKYRPPUGPPUGmhvdmAmul9B3qSEcSWIjtoT2Vs8Rg/V7kZKxshrD0srBuuhSIMaQDuAB05bKTEuTG12dDUcOsnu9Bh4YaEVgZq1smab9ZLMSi3cjuLschdfjRIPPgGE44LhnZkM6LZFKpRGOJBHdSKl+nh7SGXz6uHTBRMDpdh/a23LWJx2grk7H4EAbZoPrpm93ZpCSEY4kDhSyWw0DpwVAaus8LUIThL4zzQWLu4PYPu604nLW4wABXh1A1X9zo05HX28znEbplROn2r0wDB1z8yHL04pVhFcHUNwlUWa8nq1gSkVJamODC4ah4/bMGjbTlq59Ui14Bap4HaqWZjf6e5uV1hu7nA4MDrShvr5qv7ZKDGsqtU1CBHR1NKC7s9GSclRdFxjoa0Fzk1u57WqjPOWARaBpAn1nmuCpzzvtxhREhJ6uRrhddZhfDIO5NvvlqhLYMHT0n2mGYSKYKpaWZjfq6jTMzFUuKWIlVXOL9noMDN7XWlZx97Q90GoqSq9WqkLg1pZ69Pe2WDZ5qxCM7aRIjqmax5qKCrzTD3Z1NMDiqT0FUYtJkYoJrGsC9ymKZKVk3J5dw9JycVUPh3Gq3YvenqaCBgiqnYp0Ok6nA/1nmlFXZ37Kx+ZmFoGZVSRTGYQjSSSSafR0NZoWp1aSImW/gn1eJ+6/r1WJuLH4Jvy3lveMFoXCCUwFVpBWIIrL6cD9A61wu49vUqSsAre3ebYyUwpufavrcUwHVnNO50gk0vBPryCe2DTdjkPXcH//8U2KlEVgIsKZ7iZ0nDK/FjgzsHg3guB86MjkRDqTxdStVayFzK/OuxMMbmXWTJsrK5b3wQ5dQ19vE9wu87e5rGTMzq0XPJOAmTEXDCGRSKPztPlI/TgmRSy9gnf7MAXipjYzmJpeLmmayMpqDIGZVSWiHLekiGUCNza4toIph/lgKrqRgn96xVTpjQobO+wkRcq1dIMZLBG4vc2j7DlydS2OwMyakqvPzF1gP6LECpNyo1TgnS+tKpiaXwwjuHB0MFUsWYVJESKg45SvqpMiyjoSh0ND/5lmuFyHLxRSKJmsxMzsOjZilryIZDcST6bS6O6s7aSIkiu43l2HBwbalIibSmUwdWvFMnHvZT2UwPTtVaQzCpMiCgJKlZgW2OsxMNDfsqcAvVQi0SRuTi8jVcY65nh8czspkn/2fD4cuoaB/hbLixWKwZQqui7Q29OkpKxmZTWG27NrFal4TKezmLq1gnUFSZGtOKRJyQ9eBaa8aG81vwAnM2M2GNoumzFlSokfi3cjpm1pmkBbS3VE16bUMfscmMlITAdWlVw5qlha3th6LDN5J/FVSeGAKYHNjAglkmncnF5GLG5+QEA1kWgSU9PLSG2WHguoSPCowJTApfa94UgSU7fUDOlZRTKVgX96peCFT/ZTLc/FZY8ElpY3KhZMFUs2KxGYWVVWKVIJypoxn5sPYW29evrbQthJiqQzWXR1HL+FiMp6BVdjf1sopd6qK011PKzZWIYtcI1jC1zj2ALXOLbANY4tcI1jC1zj2ALXOLbANY4tcI1jC1zj2ALXOALA8R0BsMlHSgA4voOdNvmICgBqlie3qUaiAmBb4BqFgagg0J1KO2JjDQTcEQBuVtoRG8uYFABNVtoLG2tgwqRggi1wjcISk0Ju0usAqr+G1aZYZNrB4+L8+Y5lgN6otDc2imFMvPfB7lWx9bd8rdL+2CiG6DVgOxctBNkC1xjM2XcEjoXlqwBZ8+pMmwpA4cQGLgHbAl+82JMA+I+VdcpGFcT8uy1N7xkuZMjfVs4lG6Uw7Wq5K/DYQ93/B/Ctynhko5Cp0dGO/9/5Z1dgIpIEPFsZn2yUwfQMEe3mNfZUdLidiRfAWCi/VzaKCNa7Yv9974Y984MHBwdTV68HfwyQJVey09CrZuZ7sVTLkgxHQaBnBgcH98xzPTABPJMSP9Od/A0w+lQ70HemWbVJm20YNN3gSf1y//YDRXcXLnTGhRRfK49bNuqQ3+rv7z+wymrOqsrR0Y6/MuNl652yUcSL50a6/5xrx6Fls1mNvgkgZJlLNqpY14BvHbbzUIEvDHfOAvwE7KHEaoYB+tLISNfcYR84svB9bKT7FQDPK3fLRg1MPxkb6XzxqI/kndlQ74w/DeByrn3H5cUUlcDyc0P4h0Nb/36+j+UVeHBwMJVJJT8GYGL/PhXvP6hVkkkLzw3jhoPkY8PDw3lnpRQ0N+nChYEwsvI/Aczcu/24LWpWTlS8r+kQghmNPjY83LNWyIcLnnw2NtYzTxIfJWB2Z9t6KH6sFzezilh805IVdAmYRZYf2QqAC6Oo2YVnz3ZNprX0+xi4Bmwt83d7dg1xW+RdYvFN3J5ds2Lt6zcF8P6xse6i6thLSgzfuDHXnJbiTwAuAltvH2ludKOp0Q2ny6Hk3YTHiaxkJBNprIXiWA/F1YtL+IeQmUdHR3vXiz+0RC5dYr2pbeEHYDxtxo7NkTCA5x0i9FQhAVUuTAszfm3+USL8GkCTWVs2e4gw85fPjXb/3owR0zP8z412vSQ1OmfnrpXyogY8ZFZcQPGt9er14McBeg5Av0q7Jwe+xcCThw0clILyvvPy5TmXy6d9lZi/B6BLtf0aJUigZ32e1C9yDfmZwbLgyO/3G7GE6wtM4ikCD1jVzjFnCkzPOLT135QaROWjLNHvtWsL75HEn2PgswS0lKPN6oXCgHxZMv3PuZHOv99bIGdJa1Ya308gEHCGNhyPEItHQPwwgFHU/lJOEowJCLrEMvv3Rm/mNdW34aOo6PPrlcmFVm2Tz4HwAAHvAmMIQDvAXoAaAXgAVNfbHg+yCWAD4BBAUQB3QbjJwNssMZl28Ph7H+xerZRz/waV/uelReWF7wAAAABJRU5ErkJggg=='
```

Comment:
Is it a good idea to store an image as b64 string inside component?

Reply:
antd documentation says that image fallback is in b64....but I'll try to find some workaround solution for this

Reply:
https://github.com/open-condo-software/condo/pull/1277/commits/233eab0ae68c347c6cc5bf7a9b641fd739bd4b22

Reply:
done in child PR